### ISSUE TITLE
Crashes the database when inserting specified data

### ISSUE THREAD

**Issue by oldlwhite** [typesense/typesense#458]
~~~
import typesense
client = typesense.Client({
    'nodes': [{
        'host': 'localhost',
        'port': '8108',
        'protocol': 'http'
    }],
    'api_key': '123456',
    'connection_timeout_seconds': 2
})
schema = {
    "name": "test_data",
    "fields": [
        {"name": ".*", "type": "auto"},
    ]
}
print(client.collections.create(schema))
print(client.collections.retrieve())
data = {
    "id": "1",
    "have_file": 0,
    "kinds": {
        "CGXX": 13,
        "ZBXX": 24
    }
}
client.collections['test_data'].documents.create(data)

~~~
When the database crashes, the only way to restore the database to normal is to delete the data directory
Environment ：
typesense  0.22.0
Windows wsl  debian9
Python 3.9.1

**Comment by kishorenc** [comment]
@oldlwhite 

This indeed seems like a regression on the 0.22 release. I've a fix here, can you please test and confirm:

https://dl.typesense.org/releases/0.22.1.rc1/typesense-server-0.22.1.rc1-linux-amd64.tar.gz

**Comment by oldlwhite** [comment]
Thank you very much, this version solves the problem

**Comment by kishorenc** [comment]
Fixed in `0.22.1` patch release.
